docs/_config.yml

```diff
@@ -17,18 +17,20 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
 
 # Aux links for the upper right navigation
 aux_links:
-  'Docs':
+  'Home':
     - 'https://docs.lakefs.io/'
```

Comment:
Do we even need this link? It seems redundant. `Home` is ambiguous and could be assumed to mean lakeFS.io. I'd suggest we just drop this link.

Reply:
I agree. Removing it.